src/Angor/Client/Pages/Create.razor

```diff
@@ -782,19 +916,31 @@
 
         try
         {
-
             var words = await passwordComponent.GetWalletAsync();
             var accountInfo = storage.GetAccountInfo(network.Name);
-            var unconfirmedInfo = _cacheStorage.GetUnconfirmedInboundFunds();
+
+            unsignedTransaction = _founderTransactionActions.CreateNewProjectTransaction(
+                project.ProjectInfo.FounderKey,
+                _derivationOperations.AngorKeyToScript(project.ProjectInfo.ProjectIdentifier),
+                NetworkConfiguration.AngorCreateFeeSats,
+                project.ProjectInfo.NostrPubKey
+            );
 
             var fetchFees = await _WalletOperations.GetFeeEstimationAsync();
             feeData.FeeEstimations.Fees.Clear();
             feeData.FeeEstimations.Fees.AddRange(fetchFees);
             feeData.SelectedFeeEstimation = feeData.FeeEstimations.Fees.First();
 
-            unsignedTransaction = _founderTransactionActions.CreateNewProjectTransaction(project.ProjectInfo.FounderKey, _derivationOperations.AngorKeyToScript(project.ProjectInfo.ProjectIdentifier), NetworkConfiguration.AngorCreateFeeSats, project.ProjectInfo.NostrPubKey);
+            signedTransaction = _WalletOperations.AddInputsAndSignTransaction(
+                accountInfo.GetNextChangeReceiveAddress(),
+                unsignedTransaction,
+                words,
+                accountInfo,
+                feeData.SelectedFeeEstimation
+            );
 
-            signedTransaction = _WalletOperations.AddInputsAndSignTransaction(accountInfo.GetNextChangeReceiveAddress(), unsignedTransaction, words, accountInfo, feeData.SelectedFeeEstimation);
+            project.CreationTransactionId = signedTransaction.Transaction.GetHash().ToString();
```

Comment:
I don't think it is correct to set `CreationTransactionId` and update storage here, because we did not publish to the blockchain yet, notice we set those values later when we publish (in the method `send`) if we don't get back an error from the indexer.